tests/test_creation/test_binary_combination_feature_reference.py

```diff
@@ -0,0 +1,276 @@
+import pandas as pd
+import pytest
+
+from feature_engine.creation import BinaryCombinatioByFeatureReference
+
+
+# test param variables_to_combine
+def test_error_when_param_variables_not_entered():
```

Comment:
a bit general, wonder whether we should add a line for each obligatory param missing?